New Airflow Angles Calibration + Topics changes

acfloria opened the pull request:
- Adjust the scripts to the new firmware
- Add airflow angle calibration based on multiple loiters

acfloria committed:
Fix parameter import

acfloria committed:
Fix change in vehicle_global_position

acfloria committed:
Fix title of gps plot

acfloria committed:
Change field names to all lower case and fix verbose printing

acfloria committed:
Add alternative source for mag data in sensors plot

acfloria committed:
Move GPS logged check into plot function

acfloria committed:
Update DiffPressPlots

acfloria committed:
Update estimator plots

acfloria committed:
Update EstimatorStatus plots

acfloria committed:
Update GlobalPositionPlots

acfloria committed:
Update wind plots

acfloria committed:
Fix control plot

acfloria committed:
Fix telem rssi plots

acfloria committed:
Update cpu load plots

acfloria committed:
Fix distance sensor plot

acfloria committed:
Update mission results plot

acfloria committed:
Update vehicle status flags plot

acfloria committed:
Update mag vs thrust plot

acfloria committed:
Update power plots

acfloria committed:
Update batmon plots

acfloria committed:
Update iridium status plot

acfloria committed:
Update mppt plot

acfloria committed:
Update airflow angle plots

acfloria committed:
Adding script to calibrate the airflow angles

acfloria committed:
Adding script to generate fake data

acfloria committed:
Update airflow angle calibration

acfloria committed:
First step towards improved airflow bias calibration

- Automatically detect the loiters and optimize a bias for each loiter

acfloria committed:
Fix lever arm compensation signs

acfloria committed:
Curve fitting for the aoa and slip bias based on different loiters

acfloria committed:
Add WGS84_to_LV03 script

acfloria committed:
Add script to create an animation of multiple flight paths

acfloria committed:
Rename CalibrateAirflow to CalibrateAirflowStaticLoiter

acfloria committed:
Adding dynamic airflow calibration

acfloria committed:
Compute the std of the constant wind estimate and reconstructed wind

acfloria committed:
Adding new calibration function for the vane offsets

tstastny commented on 02_helper_functions/SolveAirflowCalibrationDynamic.m:
```diff
@@ -0,0 +1,368 @@
+end
+
+scale_factor = xopt(1);
+if config.calibration_function == 0
```
can these be enumed? or a comment at each condition at least?

tstastny reviewed (changes requested):
Oof - lot's of new stuff. I'll need to check running it myself - but maybe quicker to understand everything if we hop on a call e.g. tomorrow or some time early next week + share screen?

Would be nice in general to have a bit more comments explaining all the new parameters and configurations etc.. i was a bit lost in there.

acfloria committed:
Improve airflow angles calibration

- Add and document new calibration functions
- Rearrange code to make the addition of new functions easier
- Adding an option to to a global optimization

acfloria committed:
Adding option to force the throttle params to 0 and improve param plotting

acfloria committed:
Add output plots to airflow calibration

acfloria committed:
Support plotting either two or three flight paths in the flight path animation
